<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameLineEdit"/>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="userListWidget"/>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Message:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="messageLineEdit"/>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send this message</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="messageListWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Login"/>
    </item>
    <item>
     <widget class="QPushButton" name="banButton">
      <property name="text">
       <string>ban</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
